playbook/app/pb_kits/playbook/utilities/props.ts

```diff
@@ -47,5 +47,5 @@ export const buildDataProps = (data: {[key: string]: any}) => buildPrefixedProps
  * @param {Object} rules a 'classnames' compliant rules object, used to derive the root className.
  * @returns {String} the derived root className value.
  */
-export const buildCss = (...rules: (string | { [x: string]: string; })[]): string => classnames(rules).replace(/\s/g, '_')
+export const buildCss = (...rules: (string | { [x: string]: string | boolean; })[]): string => classnames(rules).replace(/\s/g, '_')
```

Comment:
Was there a reason for this? Not seeing it how it connects with the other two file changes.

Reply:
If I remember properly, I think he added the boolean there because {full: fullWidth} in the .tsx file was throwing a ts errorwhen he converted from .jsx to .tsx